README.md

```diff
@@ -8,37 +8,173 @@ Ballerina AWS Redshift Connector
 [![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/actions/workflows/build-with-bal-test-native.yml)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The AWS Redshift connector for [Ballerina](https://ballerina.io/) facilitates seamless interaction with Amazon Redshift data warehouses, enabling programmatic access to various applications, databases, APIs, and services through the Java Database Connectivity (JDBC) API using Ballerina. This connector empowers users to execute diverse set of standard Data Definition Language (DDL) commands, SQL commands, and SQL functions for efficient querying of data sources within the Amazon Redshift environment.
+[Amazon Redshift](https://aws.amazon.com/redshift/) is a powerful and fully-managed data warehouse service provided by Amazon Web Services (AWS), designed to efficiently analyze large datasets with high performance and scalability.
 
-For more information, go to the module(s).
-- [aws.redshift](Module.md)
+The `ballerinax/aws.redshift` connector facilitates seamless integration with Amazon Redshift, offering Ballerina users a convenient and expressive way to connect, query, and interact with Redshift clusters. 
 
-## Building from the source
+## Set up guide
```

Comment:
Module.md file needs to be updated with latest changes.

Reply:
049f312212160188bbee57611e4999dde45a5c2e